src/Button/Button.tsx

```diff
@@ -353,6 +353,12 @@ export const Button: React.FC<Props> = ({
 
                 color: getTextColor({ color, feel, theme }),
 
+                cursor: loading
+                  ? "progress"
+                  : disabled
+                  ? "not-allowed"
```

Comment:
Hmm personally I feel like we could just use `default` over `not-allowed` and `progress`. I just tried them out and they seem kind of ugly on mac at least. It might just be because not many modern websites use them, I dont think Id ever seen them before.

This is just my personal feeling so I wont request changes here, but I would be interested in hearing the reason we want them.

Reply:
No real reason. I was concerned people would not understand they can't click on disabled buttons, but I guess I'm solving a problem we don't actually have yet. The progress one is super ugly, you're right. I'll axe these.